tensorflow/lite/kernels/resize_bilinear.cc

```diff
@@ -146,7 +146,8 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
     TF_LITE_RESIZE_BILINEAR(reference_ops, ResizeBilinearInteger, int16_t);
 #undef TF_LITE_RESIZE_BILINEAR
   } else {
-    context->ReportError(context, "Output type is %d, requires float.",
+    context->ReportError(context,
```

Comment:
This is causing an internal check to fail (unfortunately not visible in the logs). TF_LITE_REPORT_ERROR should be used instead of context->ReportError, so that log strings can be removed to save space, if needed.